src/clients/cargo.js

```diff
@@ -80,7 +80,12 @@ class Cargo {
 
   #run(args, options = { stdio: 'inherit' }) {
     debug('cargo', ...args);
-    return spawn('cargo', args, { ...options, encoding: 'utf8', cwd: this.#cwd });
+    return spawn('cargo', args, {
+      ...options,
+      cwd: this.#cwd,
+      encoding: 'utf8',
+      maxBuffer: 1024 * 1024
```

Comment:
can we have a const and/or a comment to describe this magic number?

Reply:
Could be any sufficiently large number. Originally it was smaller but we increased it because there are cases when Rust debugging can produce very large output.